hasher-matcher-actioner/terraform/api/variables.tf

```diff
@@ -17,8 +17,8 @@ variable "prefix" {
   type        = string
 }
 
-variable "api_authorizer_jwt_issuer" {
-  description = "A URL to the JWT issuer (used by the api gateway authorizer)"
+variable "api_and_webapp_user_pool_id" {
+  description = "user pool id that can be used to create a URL to the JWT issuer (used by the api gateway authorizer)"
```

Comment:
Once this lands can you send in chat an example of what we should include for this variable and maybe update `terraform.tfvars.example` also

Reply:
`integration_api_access_token = "super-secret-string-I-made-by-hitting-my-head-on-the-keyboard"`

Reply:
not going to add it to `terraform.tfvars.example` because if you miss it the example value might work and give someone unintentional access